lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -274,11 +268,19 @@ private function hydrateReferenceField(
                 $this->getDefaultValue($fieldMapping)
             );
 
-            if ($referencedNode) {
-                $accessor->set(
+            try {
+                if ($referencedNode) {
+                    $accessor->set(
                     $fieldName,
-                    $this->proxyFactory->createProxyForNode($document, $referencedNode, $options)
+                    $proxyFactory->createProxyForNode($document, $referencedNode, $options)
                 );
+                }
+            } catch (\Exception $e) {
```

Comment:
What comes from the ProxyManager library? Certain exceptions? I am quite sure there is also a more specific exception for that.